Open Judge System/Data/OJS.Data/Migrations/202306120621148_StartedExecutionOn.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace OJS.Data.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class StartedExecutionOn : DbMigration
+    {
+        public override void Up()
+        {
+            AddColumn("dbo.Submissions", "StartedExecutionOn", c => c.DateTime(nullable: false));
+        }
+        
+        public override void Down()
+        {
+            DropColumn("dbo.Submissions", "StartedExecutionOn");
+        }
```

Comment:
We use Automatic migrations. Remove the migration and check how it goes without it.